src/main/io/dashboard.c

```diff
@@ -580,6 +583,33 @@ static void showDebugPage(void)
 }
 #endif
 
+static void showBBPage(void)
+{
+    uint8_t rowIndex = PAGE_TITLE_LINE_COUNT;
+    int8_t fileNo;
+    if (isBlackboxDeviceWorking()){
+        switch (blackboxConfig()->device) {
+            case BLACKBOX_DEVICE_SDCARD:
```

Comment:
Coding style in betaflight is to have the `case` statements at the same indentation level as `switch`.

Reply:
done